server/src/lib.rs

```diff
@@ -82,6 +85,14 @@ impl Settings {
     Settings {
       db_url: env::var("DATABASE_URL").expect("DATABASE_URL must be set"),
       hostname: env::var("HOSTNAME").unwrap_or("rrr".to_string()),
+      bind: env::var("BIND")
+        .unwrap_or("0.0.0.0".to_string())
+        .parse()
+        .unwrap(),
```

Comment:
Have you tested this with a real env var? Will rust unwrap a string to an `IpAddr`? I'm guessing the parse handles it fine, but double check to make sure.

Reply:
Yes, it does. See https://doc.rust-lang.org/nightly/std/net/enum.IpAddr.html#examples